### PR TITLE
RK-10907 - Remove repository settings section

### PR DESCRIPTION
**Pull request by lirook**

### CODE REVIEW

**Comment by lirook** [comment]
@eladuzan94 @ElinorSwery @jhendrick
FYI

File: docs/source-repos.md [hunk 1]
```diff
@@ -86,62 +87,3 @@ For more information, please click [here](node-setup#transpiling-and-source-maps
...
-```
-
-#### Manipulating a Path with Spaces
-
-To convert `my app/app.py` to `app.py` use:
-```bash
-"my app" /
-```
-
-### (Node) Debugging Packages
```
Comment (LiranLast) [inline docs/source-repos.md:137]: @lirook this might still needed - we don't have a browser-based workaround for it

**Comment by jhendrick** [comment]
> @eladuzan94 @ElinorSwery @jhendrick FYI

Dudi and I actually had to use this .rookout file approach a few weeks ago for a source mismatch issue on a POC call

File: docs/source-repos.md (hunk 1 above)
Comment (lirook) [reply docs/source-repos.md:137]: I'll add it to the node setup page

Comment (LiranLast) [reply docs/source-repos.md:137]: I'm not sure it belongs there.

We might want to try making smaller changes first? :)

**Comment by lirook** [comment]
> > @eladuzan94 @ElinorSwery @jhendrick FYI
> 
> Dudi and I actually had to use this .rookout file approach a few weeks ago for a source mismatch issue on a POC call

It's good that we have that workaround when needed, but I think it shouldn't be in the customer-facing docs. WDYT @jhendrick?

File: docs/node-setup.md [hunk 2]
```diff
@@ -183,3 +183,15 @@ exports.handler = rookout.wrap(handler, {token:'[Your Rookout Token]', labels:{e
 **Note:** To add the function's name automatically as a label, consider adding the following - function_name:process.env.AWS_LAMBDA_FUNCTION_NAME 
 
 For more information, please check out our [deployment-examples](deployment-examples.md).
+
+### Debugging Node Modules
```
Comment (lirook) [inline docs/node-setup.md:187]: @LiranLast
In my opinion, this fits well here. Do you not agree? :)

**Comment by jhendrick** [comment]
> > @eladuzan94 @ElinorSwery @jhendrick FYI
> 
> Dudi and I actually had to use this .rookout file approach a few weeks ago for a source mismatch issue on a POC call

Yea that's fair. It's rarely used now so as long as we have it saved that should be good.

File: docs/node-setup.md [hunk 3]
```diff
@@ -183,3 +183,15 @@ exports.handler = rookout.wrap(handler, {token:'[Your Rookout Token]', labels:{e
...
+
+By default, Rookout ignores your project's dependencies in the `node_modules` folder.
+
+If the project you wish to debug is installed as a node module, create a file in the project's repository root folder, with the following content:
+
+```
+#package
+```
```
Comment (ElDuderinos) [inline docs/node-setup.md:195]: The file needs to be named ".rookout"

**Review by ElDuderinos — CHANGES_REQUESTED** [review]

File: docs/node-setup.md (hunk 3 above)
Comment (lirook) [reply docs/node-setup.md:195]: Oops. Adding it

**Review by ElDuderinos — APPROVED** [review]

**Pull request merged**